src/shared/constants.ts

```diff
@@ -4,14 +4,14 @@ require('dotenv').config();
 export class Constants {
   static appPort = 3000;
 
-  static mongoDbName =  process.env.DB_NAME || 'timer2ticketDB';
+  static mongoDbName = process.env.DB_NAME || 'timer2ticketDB_new';
 
   static mongoDbUrl = process.env.MONGODB_URL || 'mongodb://localhost:27017';
 
-  static sentryDsn = process.env.SENTRY_DSN || '';
+  static sentryDsn = process.env.SENTRY_DSN || 'https://d9e4e19c9f15821d57796be76000be61@o4506110897750016.ingest.sentry.io/4506110899781632';
   static daysToSync = 60;
 
   static defaultWaitDurationInCaseOfTooManyRequestsInMilliseconds = 1500;
 
-  static configObjectMappingMarkedToDeleteTresholdInDays = 7;
+  static configObjectMappingMarkedToDeleteTresholdInDays = 1;
```

Comment:
Prosím změnit zpět. Ideálně to nechat jako default value pro env proměnnou.

Reply:
předěláno na env proměnnou

Reply:
Ještě prosím přidat default tedy něco jako `process.env.CONFIG_OBJECTS_DELETE_AFTER_DAYS || 7`

Reply:
done